### ISSUE TITLE
Triage ODK wiki for possible additions to Docs

### ISSUE THREAD

**Issue by adammichaelwood** [opendatakit/docs#290]
A lot of these are dev-focused, but it would be worthwhile to sift through to be sure.

https://github.com/opendatakit/opendatakit/wiki

**Comment by troublemagnet** [comment]
I guess pulling in the collect troubleshooting is a good idea.

**Comment by ankita240796** [comment]
I feel adding the following to the docs will be a good idea:
- [Aggregate App Engine troubleshooting](https://github.com/opendatakit/opendatakit/wiki/Aggregate-AppEngine-Troubleshooting)
- [Aggregate Tomcat troubleshooting](https://github.com/opendatakit/opendatakit/wiki/Aggregate-Tomcat-Troubleshooting)
- [Aggregate security note](https://github.com/opendatakit/opendatakit/wiki/Aggregate-Security-Note)
- [Collect troubleshooting](https://github.com/opendatakit/opendatakit/wiki/Collect-Troubleshooting)
- Some info on [Repository Migration guide](https://github.com/opendatakit/opendatakit/wiki/New-Repository-Migration-Guide) and a link for details in the contributors' guide

@adammichaelwood What do you think?

**Comment by lognaturel** [comment]
I feel like basically all of it should come in with some of it going to the "Developing with ODK" section, probably in a new set of pages with a sub-TOC.

A related priority is also to address #355 since we know at least one person has been confused by it.

**Comment by ankita240796** [comment]
Yes, I agree with you @lognaturel! It is definitely a better idea to pull in all of it in different sections accordingly. I will start working on this as well as on #355.
